### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by Cakeyjoy0055** [socketio/socket.io#4632] [General]
hi

**Comment by Batmanjp** [comment]
> hi

hi joy

**Reply by Cakeyjoy0055** [reply]
what is your email

**Reply by Batmanjp** [reply]
why dont you give me yours?

**Reply by Batmanjp** [reply]
heyyyy

**Reply by Cakeyjoy0055** [reply]
bc you don´t know how to make a google doc

**Reply by Batmanjp** [reply]
i have google docs dude

**Reply by Batmanjp** [reply]
il add u okay

**Reply by Batmanjp** [reply]
okay im done'

**Reply by Cakeyjoy0055** [reply]
did not get it

**Reply by Batmanjp** [reply]
fine  heres my email jhogan4577@cmsd.k12.ms.us

**Comment by darrachequesne** [comment]
Hi @Cakeyjoy0055, how can we help?
